### ISSUE TITLE
BUG | Not using the installed Python by Shuru. It is using the installed Linux distro.

### ISSUE THREAD

**Issue by airvzxf** [shuru-project/shuru#15]
# BUG

## Description

I was curious to see if it was actually taking the Python that Shuru installs, or if it was taking my general Python that comes by default on my computer. The findings led me to discover a bug where it actually uses the Python that is already installed, not the new Python.

### Set up

In the _shuru.toml_ file, I use Python version 3.10.2; on my computer, I have version 3.12.6 installed.

Adding these commands to the _shuru.toml_ file:

```toml
[versions]
python = "3.10.2"

[tasks.whereis]
command = "whereis python"

[tasks.version]
command = "python --version"
```

Local version:

```bash
$ python --version
Python 3.12.6
```

#### Expectation.

It should show me version 3.10.2, and the Python path also shows the Shuru installation.

#### Reality.

It shows version 3.12.6, and the path only shows the local one but not the Shuru one.

```bash
./shuru version
Python 3.12.6

$ ./shuru whereis
python: /usr/bin/python /usr/share/man/man1/python.1.gz
```

**Comment by Aditya-PS-05** [comment]
Hey @harshdoesdev and @airvzxf ,
Can you assign this task to me under HacktoberFest label.

**Comment by airvzxf** [comment]
> Hey @harshdoesdev and @airvzxf ,
> Can you assign this task to me under HacktoberFest label. 

The problem is solved in the issue #16. Anywhere, the owner of this project is @harshdoesdev , and he take the decisions.
